psa-crypto-sys/src/extras.rs

```diff
@@ -15,6 +15,22 @@ pub fn PSA_ALG_AEAD_TAG_TRUNCATED_LENGTH(aead_alg: psa_algorithm_t) -> usize {
     (pre_mask_tag_length & TAG_LENGTH_MASK) as usize
 }
 
+/// Retrieves the output size of an ECDH raw key agreement operation shared secret.
+/// Caller must ensure key type is compatible.
+/// This does not match any PSA macro, it will be replaces by PSA_RAW_KEY_AGREEMENT_OUTPUT_SIZE once
+/// mbedTLS adds support for it.
+pub unsafe fn PSA_RAW_ECDH_KEY_AGREEMENT_OUTPUT_SIZE(
+    _key_type: psa_key_type_t,
+    key_bits: usize,
+) -> usize {
+    /*
+    The size of the shared secret is always `ceiling(m/8)` bytes long where `m` is the bit size associated with the curve,
+    i.e. the bit size of the order of the curve's coordinate field. When m is not a multiple of 8, the byte containing the most
+    significant bit of the shared secret is padded with zero bits.
+    */
+    (key_bits - 1) / 9 // Round the division up
```

Comment:
What if `key_bits` is `0`? This might panic in debug mode and overflow in release mode. I propose the following:
```
(key_bits + 7) / 8
```
or even better using [`checked_add`](https://doc.rust-lang.org/std/primitive.usize.html#method.checked_add) for overflows. We can choose and document to return `0` if an overflow would happen.

Reply:
Didn't think of that. Though, will a valid key ever have 0 bits? I like your proposal, nice and fast 🚀